Deployment/src/Modules/FulfilmentWebapps/main.tf

```diff
@@ -39,12 +39,48 @@ resource "azurerm_app_service" "small_exchange_set_webapp" {
   https_only = true
 }
 
+resource "azurerm_app_service_slot" "small_exchange_set_staging" {
+  count               = var.exchange_set_config.SmallExchangeSetInstance
+  name                = "staging"
+  app_service_name    = azurerm_app_service.small_exchange_set_webapp[count.index].name
+  location            = azurerm_app_service.small_exchange_set_webapp[count.index].location
+  resource_group_name = azurerm_app_service.small_exchange_set_webapp[count.index].resource_group_name
+  app_service_plan_id = azurerm_app_service_plan.small_exchange_set_app_service_plan[count.index].id
+  tags                = azurerm_app_service.small_exchange_set_webapp[count.index].tags
+
+  site_config {
+    windows_fx_version  =   "DOTNETCORE|3.1"
```

Comment:
This should reference .net 6.0

Reply:
Done. Actually existing code (main branch) also has dot net core 3.1 reference. I have replaced them too.